src/Data/Fin/Base.agda

```diff
@@ -104,8 +105,8 @@ inject₁ zero    = zero
 inject₁ (suc i) = suc (inject₁ i)
 
 inject≤ : ∀ {m n} → Fin m → m ℕ.≤ n → Fin n
-inject≤ zero    (s≤s le) = zero
-inject≤ (suc i) (s≤s le) = suc (inject≤ i le)
+inject≤ {_} {suc n} zero    le = zero
+inject≤ {_} {suc n} (suc i) le = suc (inject≤ i (ℕₚ.≤-pred le))
 
```

Comment:
If these now irrelevant proofs of `m ℕ.≤ n` were marked as irrelevant,
we would get e.g. that `inject≤-irrelevant` becomes trivial.

Reply:
@gallais 
Making the proofs irrelevant introduce unsolved metas. Do you still want me to make them irrelevant? I will need to make changes to a few proofs.

Reply:
In my opinion, the fact agda couldn't solve those metas, while before it did, should be considered a bug.

Reply:
I made more arguments irrelevant and the metas were gone.